config/clock.rb

```diff
@@ -37,6 +37,12 @@ class Clock
     end
   end
 
+  every(1.day, 'UCASIntegrationCheck', at: '10:00') do
+    return unless HostingEnvironment.productioun?
+
+    UCASIntegrationCheck.perform_async if Time.zone.yesterday.weekday?
```

Comment:
Should this `if` live inside `UCASMatching::FileDownloadCheck`?

Reply:
The condition for marking a failure in `UCASMatching::FileDownloadCheck` already takes the dates into account

> elsif Time.zone.now > Time.zone.parse(last_date).next_weekday

I just don't think it's necessary to run this check on a daily basis either and followed the [patten currently used](https://github.com/DFE-Digital/apply-for-teacher-training/blob/master/config/clock.rb#L31) in the clock configuration.